drivers/sensor/bosch/bme680/bme680.c

```diff
@@ -444,12 +443,43 @@ static int bme680_init(const struct device *dev)
 		return err;
 	}
 
-	err = bme680_reg_write(dev, BME680_REG_CTRL_MEAS,
-			       BME680_CTRL_MEAS_VAL);
+	return bme680_reg_write(dev, BME680_REG_CTRL_MEAS, BME680_CTRL_MEAS_VAL);
 
 	return err;
 }
 
+static int bme680_pm_control(const struct device *dev, enum pm_device_action action)
+{
+	int rc = 0;
+
+	switch (action) {
+	case PM_DEVICE_ACTION_SUSPEND:
+	case PM_DEVICE_ACTION_RESUME:
+	case PM_DEVICE_ACTION_TURN_OFF:
```

Comment:
https://github.com/zephyrproject-rtos/zephyr/pull/75920 and https://github.com/zephyrproject-rtos/zephyr/pull/75921 do not define explict handling for `PM_DEVICE_ACTION_TURN_OFF` and instead let it return `-ENOTSUP`. Would it make sense to always implement both ON and OFF?

Reply:
Good catch, fixed